src/atopile/cli/create.py

```diff
@@ -183,6 +193,65 @@ def create(
     rich.print(f':sparkles: [green]Created new project "{name}"![/] :sparkles:')
 
 
+
+def create_build():
+    """
+    Create a new build configuration.
+    - adds entry to ato.yaml
+    - creates a new directory in layout
+    """
+    build_name = caseconverter.kebabcase(rich.prompt.Prompt.ask("Enter the build name"))
+    try:
+        top_level_path = config.get_project_dir_from_path(Path("."))
+    except FileNotFoundError:
+        raise errors.AtoError("Could not find the project directory, are you within an ato project?")
+
+    # Get user input for the entry file and module name
+    entry = rich.prompt.Prompt.ask("We will create a new ato file and add the entry to the ato.yaml,\n what would you like to call the entry file? (e.g., psuDebug)")
+
+    target_layout_path = top_level_path / "elec" / "layout" / build_name
```

Comment:
The layout path is a config param we should probably honor, or help peopel upgrade from